src/main.cpp

```diff
@@ -461,6 +461,12 @@ int main(int argc, char* argv[])
     // seed random number generator (should be done once per lifetime)
     qsrand(static_cast<quint64>(QTime::currentTime().msecsSinceStartOfDay()));
 
+    // workaround latency spikes with wifi on Qt < 5.9.4, see https://github.com/Mudlet/Mudlet/issues/1587
```

Comment:
If it is only needed for Windows and for Qt versions less than 5.9.4 then perhaps it ought to be conditional - i.e. with a:

    #if QT_VERSION < 0x050904 && defined(QT_OS_WIN32)

although IIRC the behaviour/way QT_VERSION was defined changed around 5.6, and now I think you are better off testing the version with the `QT_VERSION < QT_VERSION_CHECK(5, 9, 4)`...!

Also does this need to be further up, just above:

    auto abortLaunch = runUpdate();

as is that not where the Windows updater code starts the download (over an SSL connection) of the update and that will use the Qt networking code?

Reply:
The update is already downloaded if we need to run it.

Reply:
Added the version check wrapper. I'll keep it platform-neutral since this could have affected other platforms as well - we only know of Windows as it's the most popular one.